tests/login.go

```diff
@@ -0,0 +1,46 @@
+package tests
+
+import (
+	"fmt"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libnet"
+)
+
+// LoginToCirros performs a console login to a Cirros base VM
+func LoginToCirros(vmi *v1.VirtualMachineInstance) error {
```

Comment:
It is very confusing we have the same methods names ince under console and once under tests. I would put both on the same place and use different method names (e.g LoginToCirrosAndConfigureIpv6).

Reply:
The idea to drop these, so this is an intermediate state. If we change this name, we can just not have it at all.
Just to move things going, I am fine with waiting for the following PR to do this.

Reply:
We are keeping it like this to reduce the sice of the PR and start moving expecters out of utils.go, at a follow up PR I will do the following:
- remove tests/login.go
- create new LoginToCirrisWithIPv6 at tests/console.go.
- adapt tests to call `LoginToCirros` or `LoginToCirrosWithIPv6`.